.gitignore

```diff
@@ -3,3 +3,4 @@ lib/
 node_modules/
 *.pyc
 
+tensorflow.tar.gz
```

Comment:
Should the setup script download this to /tmp instead? I realize that this fixes this the immediate git experience, but wonder if its better to tackle at source.

Reply:
Sure it could, or to `~/.npm/_tensorflow`, similar to what `prebuild-install` does. It could also try to check if the `.tar.gz` file exists at all before trying to download it.

Reply:
I can undo the changes to `.gitignore` and just update `package.json` instead.

Reply:
Or I can make the change. I'll rebase and merge in your change, and separately address.